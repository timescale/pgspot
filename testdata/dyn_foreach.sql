create function public.fn() returns void
as $func$
declare
    _rec record;
begin
    for _rec in execute format
    ( $sql$
        select generate_series(%L, %L) as x
      $sql$
    , _catalog_id
    )
    loop
        raise notice '%', _rec.x;
        PERFORM format('%s', _rec.x);
    end loop;
end
$func$ language plpgsql volatile security invoker
;
