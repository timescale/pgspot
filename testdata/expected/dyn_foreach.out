PS005: Function without explicit search_path: public.fn() at line 1
PS016: Unqualified function call: format at line 1
PS016: Unqualified function call: format at line 1

 Errors: 0 Warnings: 3 Unknown: 0 

